setup-scripts/install.sh

```diff
@@ -81,15 +121,68 @@ msg "Installing node.js@8.11" $WHITE
 nvm install 8.11 --latest-npm
 
 if [ "$(command -v docker)" == "" ]; then
-  msg "Docker is not installed. Please install it before continuing" $RED
+  msg "Docker is not installed." $RED
+  if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    # Get the version of Linux distribution
+    LINUX_DISTRO=$(cat /etc/*-release | grep -E '^ID=' | sed 's/^.*=//' | tr -d \")
+    msg "Looks like you're using Linux." $GREEN
+    msg "To install Docker using the convenience script for Linux, run" $GREEN
+    msg "    $ curl -fsSL https://get.docker.com -o get-docker.sh" $WHITE
+    msg "    $ sudo sh get-docker.sh" $WHITE
+    msg "Alternatively, you can follow Docker's installation steps for Linux (https://docs.docker.com/install/linux/docker-ce/$(echo $LINUX_DISTRO)/)" $GREEN
+  elif [[ "$OSTYPE" == "darwin"* ]]; then
+    msg "Looks like you're using Mac." $GREEN
+    msg "You can use the following link to download and install Docker for Mac Version 2.0.0.3 or greater (this installs Docker Community Edition and Docker Compose)" $GREEN
+    msg "https://docs.docker.com/docker-for-mac/release-notes/#docker-community-edition-2003-2019-02-15" $WHITE
+  else
+    msg "Please install Docker before continuing." $GREEN
+  fi
+  exit 1
+fi
+
+# Ensure version of Docker meets minimum requirements. Warn if version is not the latest
+DOCKER_VERSION=$(docker version | grep Version | head -n 1 | grep -oE '[.0-9]*$')
+compare_versions $DOCKER_VERSION $MIN_DOCKER_VERSION
+if [[ $? == 2 ]]; then
+  msg "Your version of Docker is older than the minimum required version. Please update to the latest version of Docker." $RED
   exit 1
 fi
 
+compare_versions $DOCKER_VERSION $CURRENT_DOCKER_VERSION
+if [[ $? == 2 ]]; then
+  msg "Your version of Docker isn't up to date. Please update to the latest version." $YELLOW
```

Comment:
Maybe something like "Your version of Docker ($DOCKER_VERSION) isn't up to date. It's recommended that you upgrade to the latest version: $CURRENT_DOCKER_VERSION."

Reply:
Good call. I spoke with @dannypaz earlier about how we want to maintain `$CURRENT_DOCKER_VERSION`. We decided to manually update the variable and not include patches as opposed to using `curl` to grab the latest version. Because we're manually updating, I've changed the wording to indicate the user should get `$CURRENT_DOCKER_VERSION` or greater.